deploy/aws/README.md

```diff
@@ -39,10 +39,10 @@ This guide assumes you already have a functional Kubernetes cluster
 and helm installed (see below for instructions on installing
 helm). Note the following requirements:
 
-* The helm chart deploys Prometheus and Grafana to collect and display Triton metrics. To use this helm chart you must install Prpmetheus and Grafana in your cluster as described below and your cluster must contain sufficient CPU resourses to support these services. 
+* The helm chart deploys Prometheus and Grafana to collect and display Triton metrics. To use this helm chart you must install Prpmetheus and Grafana in your cluster as described below and your cluster must contain sufficient CPU resources to support these services.
```

Comment:
Should we keep the length below 80 characters? Can we set this with hooks?

Reply:
Great idea. We had not previously enforced this, but I updated `.clang-format` to do so and will copy the change to the other repos.

Reply:
I added this for the listed filetypes for clang-format. I did not enforce this for text or markdown files, as when I went to try running it, it looked like it broke things we don't want (e.g. the bug template's formatting).

One option could be creating a separate style configuration for text files and pointing a second clang-format pre-commit hook to that file, which enforces line length. We would do it going forward rather than fixing current long lines. That would avoid breaking anything and force us to think creatively to keep lines <80 characters in the future. I'd like to keep that out of this change though, as it's inconsistent with the way the other hooks are being run as part of the change and it increases the scope. I'll create a JIRA ticket to consider this change.